lizmap/modules/admin/locales/en_US/admin.UTF-8.properties

```
@@ -159,6 +159,7 @@ form.admin_section.message.accessControlAllowOrigin.bad.domain=Content is not a
 menu.lizmap.theme.label=Theme
 configuration.button.modify.theme.label=Modify
 form.admin_theme.h1=Theme
+form.admin_theme.helpcolor=You can select colors with this <a href="https://mdn.github.io/css-examples/tools/color-picker/">colorpicker</a> (e.g. <code>rgb(148 193 31 / 0.5)</code>).
```

```suggestion
form.admin_theme.helpcolor=You can select colors with this <a href="https://mdn.github.io/css-examples/tools/color-picker/" target="_blank">colorpicker</a> (e.g. <code>rgb(148 193 31 / 0.5)</code>).
```

otherwise, it's not convenient to leave LWC ...